app/src/debug/res/drawable/preview_seekbar_icon.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
preview_seekbar_icon.xml is in the wrong location and duplicated twice, it should be located in app/src/main.